<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cityLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgba(0, 0, 0, 0);
	font: 75 16pt &quot;Consolas&quot;;
	border:2px solid rgba(0,0,0,0);
	padding-bottom:7px;
	border-bottom-color: rgb(255,255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>City</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(47, 79, 79);
font: 75 16pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="text">
     <string>Weather</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>940</width>
      <height>770</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 169, 169);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>back.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>471</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 75 12pt &quot;Consolas&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>148</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>City</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Province</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Population</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="iconlabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>thunderstorm.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="satatuslabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 75 14pt &quot;Consolas&quot;;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="degrelabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 75 36pt &quot;Consolas&quot;;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>420</y>
      <width>134</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 15pt &quot;Consolas&quot;;
	color: rgb(255, 255, 0);
}</string>
       </property>
       <property name="text">
        <string>Population :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="provinceLabel2">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 15pt &quot;Consolas&quot;;
	color: rgb(255, 255, 0);
}</string>
       </property>
       <property name="text">
        <string>Province   :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 15pt &quot;Consolas&quot;;
	color: rgb(255, 255, 0);
}</string>
       </property>
       <property name="text">
        <string>City       :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>9</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nthbuttton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 85, 0);
	font: 75 14pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Netherland</string>
       </property>
       <property name="shortcut">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="usabutton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255, 85, 0);
	font: 75 14pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>USA</string>
       </property>
       <property name="shortcut">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trbutton">
       <property name="styleSheet">
        <string notr="true">QPushButton{

	font: 75 14pt &quot;Consolas&quot;;
	background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Turkey</string>
       </property>
       <property name="shortcut">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="citylandLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 13pt &quot;Consolas&quot;;
	color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>680</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

font: 75 16pt &quot;Consolas&quot;;
	background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="errlabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="updateDate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(255, 134, 12);
	font: 75 11pt &quot;Consolas&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="cityLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 14pt &quot;Consolas&quot;;
	color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="provinceLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 14pt &quot;Consolas&quot;;
		color: rgb(0,0,0);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="popLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 75 14pt &quot;Consolas&quot;;
	color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>700</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

  border-radius: 20px;        
    border-width: 1px;
    border-style: solid;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>info.png</normaloff>info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>cityLine</zorder>
   <zorder>weatherButton</zorder>
   <zorder>tableWidget</zorder>
   <zorder>iconlabel</zorder>
   <zorder>satatuslabel</zorder>
   <zorder>degrelabel</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>exitButton</zorder>
   <zorder>errlabel</zorder>
   <zorder>updateDate</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>aboutButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
